src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -1268,13 +1268,14 @@ private SynthesizedRecordConstructor TryGetSynthesizedRecordConstructor(RecordDe
             return symbol;
         }
 
-        private AttributeSemanticModel CreateModelForAttribute(Binder enclosingBinder, AttributeSyntax attribute, MemberSemanticModel containingModel)
+#nullable enable
+        private AttributeSemanticModel CreateModelForAttribute(Binder enclosingBinder, AttributeSyntax attribute, MemberSemanticModel? containingModel)
         {
             AliasSymbol aliasOpt;
             var attributeType = (NamedTypeSymbol)enclosingBinder.BindType(attribute.Name, BindingDiagnosticBag.Discarded, out aliasOpt).Type;
 
             // For attributes where a nameof could introduce some type parameters, we need to track the attribute target
-            Symbol attributeTarget = getAttributeTarget(attribute.Parent.Parent);
+            Symbol? attributeTarget = getAttributeTarget(attribute.Parent?.Parent);
```

Comment:
Adding these `?.`s everywhere we access `Parent` is likely going to invalidate some invariant somewhere. I think we need to get an understanding of the syntax that is causing a null reference before making more changes.

Reply:
Unfortunately I lost the repro, and the particular syntax that set this off wasn't clear in the first place. Given that I'll close this and try again in a more targeted fashion the next time I see it.